sunpy/net/base_client.py

```diff
@@ -203,7 +203,9 @@ def _print_client(client, html=False):
     if html:
         lines = [f"<p>{line}</p>" for line in lines]
     lines.extend(t.pformat_all(show_dtype=False, max_width=width, align="<", html=html))
-    return '\n'.join(lines)
+    if visible_entries is None:
+        return '\n'.join(lines)
+    return '\n'.join(lines[slice(visible_entries)])
```

Comment:
```suggestion
    return '\n'.join(lines[slice(0,visible_entries)])
```

Reply:
This should be the correct suggestion now.

Reply:
Sorry about that.

Reply:
No worries, I have made the changes. I am sorry if I bothered you too much with this pull request, I will try to be more careful from now onwards.

Reply:
No, that's what we are here for.